app/views/devise/sessions/new.html.erb

```diff
@@ -1,29 +1,21 @@
-<div class="form-container">
-<h2>Log in</h2>
- 
-<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
-  <div class="field">
-    <%= f.label :email %><br />
-    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
-  </div>
-
-  <div class="field">
-    <%= f.label :password %><br />
-    <%= f.password_field :password, autocomplete: "current-password" %>
-  </div>
-
-  <% if devise_mapping.rememberable? %>
-    <div class="field">
-      <%= f.check_box :remember_me %>
-      <%= f.label :remember_me %>
+<%= form_for(resource, as: resource_name, url: session_path(resource_name), html: { data: { turbo: false} }) do |f| %>
+  <div class="card container" style="padding: 3rem; max-width: 700px; margin: 1rem auto;">
+    <h2 class="text-center" style="margin: 0 0 1rem 0;">Log in</h2>
+    <div class="mb-3">
+      <%= f.email_field :email, autofocus: true, autocomplete: "email", class: 'form-control', placeholder:'Email'  %>
+    </div>
+    <div class="mb-3">
+      <%= f.password_field :password, autocomplete: "current-password", class: 'form-control', placeholder:'Password' %>
+    </div>
+    <% if devise_mapping.rememberable? %>
+      <div class="form-check form-switch">
+        <%= f.check_box :remember_me, class:'form-check-input', id:"flexSwitchCheckDefault", role:"switch" %>
+        <%= f.label :remember_me, for:'flexSwitchCheckDefault', class:"form-check-label" %>
+      </div>
+    <% end %>
+    <div class="d-grid gap-2">
+      <%= f.submit "Log in", class:"btn btn-outline-info" %>
     </div>
```

Comment:
- [ ] Kindly link the first page to the splash screen as requested by the previous reviewer. Users should choose the pages to go to when they visit the app.


![splash](https://github.com/Nasirkhan294/penny_planner/assets/53124548/3d5ca451-9afc-4dfc-bcbd-2e8aacfab477)


![splash-screen](https://github.com/Nasirkhan294/penny_planner/assets/53124548/0f05842d-ca40-44e6-882e-2a4c89d10d5c)

Reply:
Hi @elisha2kyakpo1, I have added the code but I don't what's wrong with it and why it's not showing the splash screen. Kindly, suggest me tips to overcome this problem.